### PR TITLE
adding file

### PR DESCRIPTION
**Pull request by manimanjari**
it takes a movie name and gets imdv rating

### CODE REVIEW

File: try2.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+#Download the result page for the topmost search
+curl "http://www.imdb.com$(head -n 1 filesToDownload.txt)" -o "datafile.html" 
+
+# run regex and extract the division containg the imdb rating
+sed -e '/itemprop="ratingValue"/,/findMoreMatches/!d' "datafile.html" > "rating.txt" 
+
+#chop the div to get the desired rating and append it to final file
+awk '{for(i=1;i<=NF;i++) if ($i=="based")  print $(i-1)}' rating.txt | cut -c8-10 >> "answer.txt" 
+
```
Comment (vaidik) [inline try2.sh]: Too many blank lines at the end. At max one is allowed

File: try2.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+# Download the result page for the specified search
+curl "http://www.imdb.com/find?q=$searchText" -o "$searchText-search.html" 
+
+# Define the file where the links will be stored
+writeLinksFileName="filesToDownload.txt"
+
+# Run regex to get the movie titles div and grep data to extract the title code
+sed -e '/Titles<\/h3>/,/findMoreMatches/!d' "$searchText-search.html" | grep -E -w -o "\/title\/[a-zA-Z0-9]+\/"> $writeLinksFileName
```
Comment (vishesh92) [inline try2.sh]: https://google.github.io/styleguide/shell.xml#Line_Length_and_Long_Strings

File: try2.sh [hunk 3]
```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Read the movie name
+read searchText
+
+#appends it to a final file
```
Comment (vishesh92) [inline try2.sh]: All comments should be on the same level of indentation as the code it refers to. All comments must start with # (i.e. a hash symbol and a space).

**Comment by vaidik** [comment]
```
$ bash try2.sh
inception
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 43796    0 43796    0     0  21675      0 --:--:--  0:00:02 --:--:-- 21681
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  197k    0  197k    0     0  78029      0 --:--:--  0:00:02 --:--:-- 78020
➜  /tmp cat answer.txt
inception
8.8
```

**Comment by vaidik** [comment]
When I executed your script, I got the following output. The first thing - what is that useless output in between? That does not matter to me. Can we get rid of it? (Hint: stream redirection)

Second thing - the script did not suggest that I have to read the output from answer.txt. Ideally the output should be printed by the script itself.

**Comment by vaidik** [comment]
I executed your script with `Inception` as the first input. The next time when I executed with `Thor`, and printed `answer.txt`, this is what I got:

```
inception
8.8
thor
7.0
```

Notice the left over results from the previous run?

**Pull request merged**